src/views/accounts/VirtualAccount/VirtualList.vue

```diff
@@ -22,7 +27,15 @@ export default {
             name: {
               sortable: false,
               formatterArgs: {
-                route: 'VirtualAccountDetail'
+                drawer: true,
+                getRoute: ({ row }) => {
+                  return {
+                    name: 'VirtualAccountDetail',
+                    params: {
+                      id: row.id
+                    }
+                  }
+                }
               }
             },
             actions: {
```

Comment:
Unfortunately, you did not provide any code to analyze for checking differences and potential issues/optimization suggestions. Please share the relevant code so I can assist with your request.